website/docs/r/one_dashboard.html.markdown

```diff
@@ -357,7 +368,62 @@ Each widget type supports an additional set of arguments:
     * `initial_sorting` - (Optional) An attribute that describes the sorting mechanism for the table. This attribute requires specifying the following attributes in a nested block -
         * `name` - (Required) The name of column to be sorted. Examples of few valid values are `timestamp`, `appId`, `appName`, etc.
         * `direction` - (Required) Defines the sort order. Accepted values are `asc` for ascending or `desc` for descending.
+    * `data_format` - (Optional) A nested block that describes data format. See [Nested data_format blocks](#nested-data_format-blocks) below for details.
+        
+
+### Nested `data_format` blocks
+
+Nested `data_format` specifies the data structure format in the table. It defines how the data is organized and interpreted.This attribute requires specifying the following attributes in a nested block -
+
+  * `name` - (Required) This attribute mandates the specification of the column name to be formatted. It identifies which column the data format should be applied to.
+  * `type` - (Required) This attribute sets the format category for your data. Accepted values include `decimal` for numeric values, `date` for date/time values, `duration` for length of time, `recent-relative` for values referencing a relative point in time, `custom` for specific date/time formats and `humanized` to set autoformat as true.
+  * `format` - (Optional) This attribute is provided when the `type` attribute is set to `date`, defining the specific date format to be applied to your data.
```

Comment:
```suggestion
  * `format` - (Optional) This attribute is provided when the `name` is that of a column comprising date/time values and the `type` attribute is set to `custom`.......
```

probably incorrect unless I'm mistaken (apologies if so) but I think `format` works only with `custom` as the type, right?